DQMOffline/Muon/src/DiMuonHistograms.cc

```diff
@@ -190,6 +194,42 @@ void DiMuonHistograms::analyze(const edm::Event & iEvent,const edm::EventSetup&
 	    }
 	  }
 	}
+	// Also Medium-Medium Muon Selection
+	if ( muon::isMediumMuon(*muon1)  && 
+	     muon::isMediumMuon(*muon2) ) { 
+	  test->Fill(InvMass);
+	  LogTrace(metname)<<"[DiMuonHistograms] Medium-Medium pair"<<endl;
+	  if (charge < 0){
+	    for (unsigned int iEtaRegion=0; iEtaRegion<3; iEtaRegion++){
+	      if (iEtaRegion==0) {EtaCutMin= 0.;         EtaCutMax=2.4;       }
+	      if (iEtaRegion==1) {EtaCutMin= etaBMin;    EtaCutMax=etaBMax;   } 
+	      if (iEtaRegion==2) {EtaCutMin= etaECMin;   EtaCutMax=etaECMax;  }
+	      
+	      if(fabs(recoCombinedGlbTrack1->eta())>EtaCutMin && fabs(recoCombinedGlbTrack1->eta())<EtaCutMax && 
+		 fabs(recoCombinedGlbTrack2->eta())>EtaCutMin && fabs(recoCombinedGlbTrack2->eta())<EtaCutMax){
+		if (InvMass > 55. && InvMass < 125.) MediumMediumMuon[iEtaRegion]->Fill(InvMass);
+	      }
+	    }
+	  }
+	}
+	// Also Loose-Loose Muon Selection
+	if ( muon::isLooseMuon(*muon1)  && 
+	     muon::isLooseMuon(*muon2) ) { 
+	  test->Fill(InvMass);
+	  LogTrace(metname)<<"[DiMuonHistograms] Loose-Loose pair"<<endl;
+	  if (charge < 0){
+	    for (unsigned int iEtaRegion=0; iEtaRegion<3; iEtaRegion++){
+	      if (iEtaRegion==0) {EtaCutMin= 0.;         EtaCutMax=2.4;       }
```

Comment:
please declare the etacutMin/Max vales  in const arrays to avoid the needless if statements